### DISCUSSION TITLE
Options for limiting Stack Size upgrades

### DISCUSSION THREAD

**Discussion by brisingraerowing** [P3pp3rF1y/SophisticatedBackpacks#673] [Ideas]
In a Create based pack I'm working on, I'd like to limit the stack size upgrades to 1 of any type per backpack, for balancing reasons. I'd also like this for Sophisticated Storage (I'll put a discussion on that repo if you want).

**Comment by Patryk122PL** [comment]
Yes! Exactly!
Configuration would be really awesome, because now in my modpack I had to disable stack upgrades completly..

Just look at this and tell me if this is "balanced"?:

Almost 17M multiplier ... that's like over milliard (10^9) items PER SLOT!
![2023-01-19_15 48 08](https://user-images.githubusercontent.com/9610133/213474652-1b6a0a5e-36ee-4925-9e4f-83adb2f38175.png)

Edit:
Also as a side note.. I nerfed netherite backpack a little bit (6 upgrades slots, instead of 7).. So "out of the box" the multiplier would be even bigger..

I could limit upgrade slots, but then what's the point of such cool upgrades if i can only have let's say 2 at the same time to keep it balanced?

Edit 2:
My suggestion is to add configuration options and / or change this multiplier to be additive not multiplicative.
